src/theme/typography.ts

```diff
@@ -0,0 +1,3 @@
+const typography = {};
```

Comment:
you have to specify types for typography object, to know it's type you need to go theme file with createTheme function, press command and click on typography key

Reply:
resolved